src/pages/PlanPage/subRoutes/CurrentOrgPlan/CurrentOrgPlan.tsx

```diff
@@ -75,4 +77,18 @@ function CurrentOrgPlan() {
   )
 }
 
+const DelinquentAlert = () => {
+  return (
+    <>
+      <Alert variant={'error'}>
+        <Alert.Title>Your most recent payment failed</Alert.Title>
+        <Alert.Description>
+          Please try a different card or contact support at support@codecov.io
```

Comment:
nit - do we need a period on this sentence?

Reply:
Good call! missed that from designs